src/components/card/event-card/index.style.js

```diff
@@ -0,0 +1,107 @@
+import styled from 'styled-components';
+import { Link as ReactRouterLink } from 'react-router-dom';
+
+import CardTemplate from '../card-template';
+import { Text, Heading } from '../../typography';
+import Icon from '../../icons';
+
+
+/* Common event card styles */
+
+export const Link = styled(ReactRouterLink)`
+  text-decoration: none;
+`;
+
+export const DateBookmarkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+export const BookmarkIcon = styled(Icon)`
+  margin-left: 0;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export const Title = styled(Heading)`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const PlacePriceIcon = styled(Icon)`
+  margin-right: 0;
```

Comment:
if we use so many icons with margin-left/right as 0 value, we can set it as a prop for icon element, so it's no needed to set a different styled for each icon every time we use it, as we have this kind of style in this file 2 times.

It makes code(bigger components that use icons with this styles) easier to read and maintain.